svg/elements/feMergeNode.json

```diff
@@ -0,0 +1,101 @@
+{
+  "svg": {
+    "elements": {
+      "feMergeNode": {
+        "__compat": {
+          "mdn_url": "https://developer.mozilla.org/docs/Web/SVG/Element/feMergeNode",
+          "support": {
```

Comment:
`<feMerge>` doesn't work without `<feMergeNode>`. So, since there is a bit more detailed compatibility info for `<feMerge>`, I think you should just copy the general support info over from there.